presto-main/src/main/java/com/facebook/presto/ExceededMemoryLimitException.java

```diff
@@ -52,6 +53,12 @@ public static ExceededMemoryLimitException exceededLocalTotalMemoryLimit(DataSiz
                 format("Query exceeded per-node total memory limit of %s [%s]", maxMemory, additionalFailureInfo));
     }
 
+    public static ExceededMemoryLimitException exceededLocalRevocableMemoryLimit(DataSize maxMemory, String additionalFailureInfo)
+    {
+        return new ExceededMemoryLimitException(EXCEEDED_REVOCABLE_MEMORY_LIMIT,
```

Comment:
put the first param to its own line. Unfortunately, the other part of the code followed our old coding style.